### DISCUSSION TITLE
Effect of phase space sample size on polarization uncertainties

### DISCUSSION THREAD

**Discussion by redeboer** [ComPWA/polarimetry#237] [General]
As a follow-up to #231 and #232, here's a comparison of the polarization $\vec{P}$ as determined from the field $\vec\alpha(\tau)$ and the averaged polarimeter $\overline{\alpha}$:

| Phsp size | from $\vec\alpha(\tau)$ | from $\overline{\alpha}$ | Distribution |
|---:|:---:|:---:|:---:|
|   **100.000** |  [![1e5](https://user-images.githubusercontent.com/29308176/197726473-1c91791d-6ffc-40fc-a0cd-a5d531e07707.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378374/artifacts/file/public/zz.polarization-fit.html) |  [![1e5](https://user-images.githubusercontent.com/29308176/197727537-25ccc690-4242-40d1-8251-125ef0fa98bf.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378374/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378374/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197728648-8fb21b2f-3556-439b-9f6d-3bf2fa568dcc.svg" width="300px"></a>
|   **200.000** |  [![2e5](https://user-images.githubusercontent.com/29308176/197726478-ee73c1e6-dbc8-4b9f-a4cc-c1f3c38581b6.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378375/artifacts/file/public/zz.polarization-fit.html) |  [![2e5](https://user-images.githubusercontent.com/29308176/197727541-6cf3f882-6cf5-4d52-8a22-b7405361787c.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378375/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378375/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197728651-7390ee19-3de0-45cd-a087-0e619982b4cf.svg" width="300px"></a>
|   **400.000** |  [![4e5](https://user-images.githubusercontent.com/29308176/197726483-30ad7650-8e6b-4fcf-ba73-97cbfa6e3826.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378382/artifacts/file/public/zz.polarization-fit.html) |  [![4e5](https://user-images.githubusercontent.com/29308176/197727545-3bca970e-7dea-4fa1-8449-f91e5989827c.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378382/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378382/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197728653-1ea25c1e-a5b5-4d70-b774-3a53b2186636.svg" width="300px"></a>
|   **800.000** |  [![8e5](https://user-images.githubusercontent.com/29308176/197726487-e447c7cc-00b8-4c58-85ca-2315fb7fc1c5.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378387/artifacts/file/public/zz.polarization-fit.html) |  [![8e5](https://user-images.githubusercontent.com/29308176/197727546-4e7f95cd-eb62-4624-86c6-1e213a0f1161.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378387/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378387/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197728655-403725e3-22ab-4515-a50d-e98c2d2e3ee7.svg" width="300px"></a>
| **1.600.000** | [![16e5](https://user-images.githubusercontent.com/29308176/197726492-19a22cf1-3469-4dc6-8278-9a6c72196691.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378391/artifacts/file/public/zz.polarization-fit.html) | [![16e5](https://user-images.githubusercontent.com/29308176/197727547-b28d6c1a-a05e-4e7c-8361-42d5ba1fd880.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378391/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25378391/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197728661-a4c66740-e8e6-4202-8cd8-8b84db4feffc.svg" width="300px"></a>

**Comment by mmikhasenko** [comment]
Hi Remco, well done! I thought exactly the same on the way home: we face the stat error of the toy sample.

The hypothesis was that,
We went around it when fitting with the same method (pretty much by chance), but changing the fitting approach this event_size fluctuation came back.

What you have done is the best way to demonstrate it. Alternatively one could do 1000 toy sample of the same size to see RMS related to the statistics.

**Reply by redeboer** [reply]
Ahh yes, that's it!
The nominal values are also closer to each other for larger sample sizes (apart from $P_y$). At the time of posting, I was only considering the uncertainties.

**Comment by redeboer** [comment]
Here's the same exercise, but setting a different seed, and using a phase space sample of 1.000.000.

| Seed | from $\vec\alpha(\tau)$ | from $\overline{\alpha}$ | Distribution |
|:---:|:---:|:---:|:---:|
| **0** |  [![seed0](https://user-images.githubusercontent.com/29308176/197739829-cd9f1266-7288-4871-aea5-d290b7622cfa.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385059/artifacts/file/public/zz.polarization-fit.html) |  [![seed0](https://user-images.githubusercontent.com/29308176/197737980-eb821588-9b04-4050-a679-e238ac5e62cf.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385059/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385059/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197738760-518d50c8-664c-4be8-b1db-61bfe3ab10de.png" width="300px"></a>
| **1** |  [![seed1](https://user-images.githubusercontent.com/29308176/197739836-c1b4b543-b489-4fd3-8f88-bb8448c0e626.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385071/artifacts/file/public/zz.polarization-fit.html) |  [![seed1](https://user-images.githubusercontent.com/29308176/197737984-320f5967-5436-4c46-8611-b2080154158b.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385071/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385071/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197738762-51553be1-4408-4f3e-969b-a8ac46b2b27b.png" width="300px"></a>
| **2** |  [![seed2](https://user-images.githubusercontent.com/29308176/197739838-de186a51-f6e8-4395-a609-ec1c43158c15.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385109/artifacts/file/public/zz.polarization-fit.html) |  [![seed2](https://user-images.githubusercontent.com/29308176/197737988-853e9a2d-0198-4112-8efb-a9c99a0d6de7.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385109/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385109/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197738764-33ab72ca-e3e4-4913-bcff-aaf4706fdffc.png" width="300px"></a>
| **3** |  [![seed3](https://user-images.githubusercontent.com/29308176/197739842-2dc723f4-5c09-46fa-abfa-4537dc0be9db.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385110/artifacts/file/public/zz.polarization-fit.html) |  [![seed3](https://user-images.githubusercontent.com/29308176/197737990-b90eb5c2-475b-4117-a7dc-a1448d8d41e4.svg)](https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385110/artifacts/file/public/zz.polarization-fit.html) | <a href="https://gitlab.cern.ch/polarimetry/Lc2pKpi-preview/-/jobs/25385110/artifacts/file/public/_static/images/polarization-distribution-systematics.svg"><img src="https://user-images.githubusercontent.com/29308176/197738766-44e9f703-c40e-4e62-8793-ec2c8b84a540.png" width="300px"></a>

**Reply by mmikhasenko** [reply]
yea, it indicates the same effect, it seems.

**Reply by redeboer** [reply]
What to conclude from it? It seems to me that one cannot reliably estimate $P_y$ in particular without doing some kind of phase space bootstrapping. So just using the polarimeter fields/averages as done here does not suffice.
